docs/contributing.md

```diff
@@ -18,6 +18,39 @@
 >
 > `workspaceName` in these commands is the `name` field within a package's `package.json`, e.g., `@metamask/address-book-controller`, not the directory where it is located, e.g., `packages/address-book-controller`.
```

Comment:
Nit: Is this note now redundant?

Reply:
Hmm, do you mean that we shouldn't need to explain what a workspace name is now?

Reply:
I thought it was a repeat of what you added below on line 36. Though now I see that it's not exactly a repeat (it's making the same clarification but regarding two different subjects), so you can disregard this comment.